integration/optimize_memory_test.go

```diff
@@ -59,7 +59,6 @@ func testOptimizeMemory(t *testing.T, context spec.G, it spec.S) {
 				settings.Buildpacks.Processes.Online,
 			).
 			WithEnv(map[string]string{"BP_NODE_OPTIMIZE_MEMORY": "true"}).
-			WithNetwork("host").
```

Comment:
@pacostas is this change and the other similar ones intended in this PR or does it need a rebase?

Reply:
network host is necessary when we fetch the builder from local registry. In our case the builder is published in a "normal" registry so this option is not necessary

Reply:
Ack, thanks for confirming